docs/spring-boot/core-starter.md

```diff
@@ -124,6 +124,42 @@ public class HelloBot {
 
 You can finally run your Spring Boot application and verify that your bot always replies with `Hello!`. 
 
+### OBO (On behalf of) usecases
+It is possible to run an application with no bot service account configured in order to accommodate OBO usecases only.
+For instance the following configuration is valid:
+```yaml
+bdk:
+    host: acme.symphony.com
+    app:
+      appId: app-id
+      privateKey:
+        path: /path/to/rsa/privatekey.pem
+```
+
+This will cause all features related to the datafeed loop such as Real Time Events, activities, slash commands, etc. to be deactivated.
+However, service beans with OBO-enabled endpoints will be available and can be used as following:
+```java
+@Component
+public class OboUsecase {
+
+  @Autowired
+  private MessageService messageService;
+
+  @Autowired
+  private OboAuthenticator oboAuthenticator;
+
+  public void doStuff() {
+      final AuthSession oboSession = oboAuthenticator.authenticateByUsername("user.name");
+      final OboMessageService obo = messageService.obo(oboSession);
+      obo.send("stream.id", "Hello from OBO"); // works
```

Comment:
I'd simply to `final OboMessageService obo = messageService.obo(oboSession).send("stream.id", "Hello from OBO");`